src/lang/index.js

```diff
@@ -21,3 +22,5 @@ export const messages = {
   'zh-CN': zh_CN,
   'ja-JA': ja_JA,
 };
+// enable i18n feature flag
+export const enablei18n = getSetting(['features', 'docs', 'i18n', 'enable'], false);
```

Comment:
Going to merge as-is, but I'm not 100% this updated placement of the `getSetting` call is working as you expect.